crates/subspace-archiving/src/archiver.rs

```diff
@@ -247,6 +262,21 @@ pub struct Archiver {
     last_archived_block: LastArchivedBlock,
 }
 
+// Equality, mainly for use in integration tests
```

Comment:
Semantically this doesn't really make a lot of sense IMO, which is why tests did not use it. There should be no need and it does not make sense to compare two archiver instances.